### ISSUE TITLE
Ecommerce campaign name view is not using the selected time range

### ISSUE THREAD

**Issue by pwagner** [matomo-org/plugin-MarketingCampaignsReporting#12]
In Ecommerce > Sales, there is a view labeled "campaign names".
There, the data does not seem to take the selected time range into account.
It looks like it is omitting all data that is older than the current month.

We are running the Piwik 3.0.4 and version 3.0.2 of this plugin.

**Comment by andre-hh** [comment]
I've just checked that. The only report AOM is providing is the "Marketing Performance" report under "Referrers". We have not modified any existing reports, i.e. AOM should not have any effect on the report you mention. 

Are you sure that this is an issue related to AOM? Perhaps it is a general Piwik issue?

For now, we do not plan to modify the existing Piwik reports, i.e. you will not be able to see the AOM campaign structure in them.

**Comment by mattab** [comment]
Are you using https://plugins.piwik.org/MarketingCampaignsReporting ?

Could you post a screenshot showing the problem somehow?

**Comment by pwagner** [comment]
Yes, MarketingCampaignsReporting v3.0.2 is installed and active.

I made a gif to illustrate the problem:
![Campaign names time range issue](http://i.imgur.com/ckDtt14.gif)

It starts under Ecommerce > Sales with the default selection of Referrer Type. The time range is set to "year". The Campaigns-row shows 25 Ecommerce orders.
Next I clicked on Campaign Names. There, the sum of ecommerce orders amounts to 12, not 25.
Then I change the time range to "month". Piwik automatically goes back to Referrer Type. There, campaign ecommerce orders are 13 (which is much closer to 12 than 25).
Finally I select Campaign Names again, and basically I see the same data as before, when "year" was the selected time range.
That's why I think the time range in the Campaign Name report is not working right.

**Comment by pwagner** [comment]
Since September, the issue no longer exists for data from August. Apparently the data prior to August somehow got corrupted in our database. Unfortunately I cannot explain it, but it seems to be working fine from now on.

**Comment by snake14** [comment]
Closing. I'm not sure why the automation reopened this issue.
